Need help! How can I calculate the height?

In the last version I use:

`const strArr = this.doc.splitTextToSize(data.dunningCustomers[i].text, 170);
 const dim = this.doc.getTextDimensions(strArr);
 const heigth = Math.ceil(dim.h);`

How can I calculate in the current version how high this array is printed?

Thanks Ingo

Hi,
If I enter it like that, I get the following error message when I call ng serve.
ERROR in src/app/invoice/services/invoice-print.service.ts:117:46 - error TS2339: Property 'h' does not exist on type '{ x: number; y: number; }'.

117                 const heigth = Math.ceil(dim.h) - 8;

--- a/types/index.d.ts
+++ b/types/index.d.ts
@@ -915,7 +915,16 @@ declare module "jspdf" {
     setHeaderFunction(
       func: (jsPDFInstance: jsPDF, pages: number) => number[]
     ): jsPDF;
-    getTextDimensions(txt: string, options?: any): { x: number; y: number };
+    getTextDimensions(
+      text: string,
+      options?: {
+        font?: string;
+        fontSize?: number;
+        maxWidth?: number;
+        scaleFactor?: number;
+      }
+    ): { w: number; h: number };
+
     cellAddPage(): jsPDF;
     cell(
       x: number,

